ciel/common.py

```diff
@@ -37,8 +37,8 @@ def date_from_iso8601(string: str) -> datetime:
     return datetime.strptime(string, ISO8601_FMT)
 
 
-def mkdirp(path):
-    return pathlib.Path(path).mkdir(parents=True, exist_ok=True)
+def mkdirp(path: str) ->None: 
+pathlib.Path(path).mkdir(parents=True, exist_ok=True)
```

Comment:
While removing the `return` is a good idea for clarity, you also removed the indentation, making this invalid Python code. Did you test this file?